subprojects/docs/src/docs/userguide/jvm/building_java_projects.adoc

```diff
@@ -276,20 +282,19 @@ To use Java 6 or Java 7, the following tasks need to be configured:
 * `Javadoc` task to use the correct `javadoc` executable
 * `Test` and the `JavaExec` task to use the correct `java` executable.
 
-The following sample shows how the `build.gradle` needs to be adjusted. In order to be able to make the build machine-independent, the location of the old Java home and target version should be configured in `GRADLE_USER_HOME/gradle.properties` footnote:[For more details on `gradle.properties` see <<build_environment.adoc#sec:gradle_configuration_properties,Gradle configuration properties>> ] in the user's home directory on each developer machine, as shown in the example.
+With the usage of Java toolchains, this is now a trivial thing to do:
```

Comment:
Let's avoid referring to recency in the docs (saying things "this is now"). Unlike release notes which are meant to highlight the news, the docs are versioned per release.